site/docs/design_tokens/border-radius.mdx

```diff
@@ -0,0 +1,51 @@
+---
+name: Border Radius
+menu: Design tokens
+route: /design-tokens/border-radius
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+
+# Border Radius
+
+<LargeParagraph>The border-radius tokens are used to give elements rounded corners.</LargeParagraph>
+
+The border-radius CSS property rounds the corners of an element's outer edge. The radius applies to the background element, even if the element has no border.
+
+## Principles
+
+When creating components or editing existing ones, keep in mind, that rounded corners, whether small or big, **should not be applied to any part of the Helsinki brand assets**. 
```

Comment:
Maybe
Rounded corners are only reserved to specific user interface needs. Rounded corners should not be applied just for styling effect and should not be applied in brand assets.

Reply:
👍